_episodes/17-live.md

```diff
@@ -37,7 +37,7 @@ Importantly, learners are strongly encouraged
 to "code-along" with the instructor.
 We refer to the practice of having the instructor live code and the learners code along as "participatory live coding" or, less formally, 'code-along sessions'.
 
-> ## Up and Down
+> ## Anticipate the Pros and Cons
```

Comment:
```suggestion
> ## Anticipate the Impact
```
maybe this? 
inspired by @ndporter 's [comment](https://github.com/carpentries/instructor-training/pull/1426#issuecomment-1233277776) and @jonathanwheeler01 's initial suggestion.